src/impl/Formulas.ts

```diff
@@ -37,10 +37,10 @@ export function byIndex<T>(
     return new FormulaCell((fromVal: T[]) => fromVal[index], source)
 }
 
-export function toBool(source: Cell<unknown>): Value<boolean> {
+export function toBool(source: Cell<unknown>): FormulaCell<boolean> {
```

Comment:
We probably should use the `formula` function instead of creation of a `FormulaCell` object. This way we can use these functions with `Value` and return `Value` as well

Reply:
Those "prepared formulas" (`not`, `toBool`, etc...) now use all `formula` function instead of `new FormulaCall`.  I also added more type tests to ensure returned types for various input types.

How about adding a GitHub action to automatically run tests on each commit/pull request?

Reply:
Yes, this sounds as a good idea